cdc/entry/codec.go

```diff
@@ -31,14 +31,15 @@ import (
 var (
 	tablePrefix  = []byte{'t'}
 	recordPrefix = []byte("_r")
-	metaPrefix   = []byte("m")
+	// MetaPrefix is the prefix for meta key.
+	MetaPrefix = []byte("m")
```

Comment:
I seems that `MetaPrefix` should not be public. It would result in tightly coupled packages if used inappropriately.